### DISCUSSION TITLE
DIY Smart Electric Toothbrush using ESP32-C2 and RainMaker · Espressif Developer Portal

### DISCUSSION THREAD

**Discussion by giscus** [espressif/developer-portal#209] [General]
# DIY Smart Electric Toothbrush using ESP32-C2 and RainMaker · Espressif Developer Portal

The developer resources in just one place!

https://developer.espressif.com/blog/esp32c2-smart-electric-toothbrush/

**Comment by ShakhawatFahim-THBD** [comment]
Hi.
A great project. Is there more info available for the firmware and the design files? Can we recreate it?

**Reply by ramkumarkoppu** [reply]
Is this reference design available as development kit?

**Reply by pedrominatel** [reply]
> Is this reference design available as development kit?

Hi @ramkumarkoppu,
For now, we do not have plans for releasing the reference design as a development kit.
Thank you for your interest.
